services/rfq/relayer/reldb/base/model.go

```diff
@@ -124,16 +138,21 @@ func FromQuoteRequest(request reldb.QuoteRequest) RequestForQuote {
 		OriginTokenDecimals:  request.OriginTokenDecimals,
 		OriginTxHash:         stringToNullString(request.OriginTxHash.String()),
 		RawRequest:           hexutil.Encode(request.RawRequest),
-		SendChainGas:         request.Transaction.SendChainGas,
+		ExclusivityRelayer:   request.Transaction.ExclusivityRelayer.String(),
+		ExclusivityEndTime:   time.Unix(int64(request.Transaction.ExclusivityEndTime.Uint64()), 0),
 		DestTokenDecimals:    request.DestTokenDecimals,
 		DestToken:            request.Transaction.DestToken.String(),
 		DestTxHash:           stringToNullString(request.DestTxHash.String()),
 		OriginAmountOriginal: request.Transaction.OriginAmount.String(),
 		OriginAmount:         decimal.NewFromBigInt(request.Transaction.OriginAmount, int32(request.OriginTokenDecimals)),
 		DestAmountOriginal:   request.Transaction.DestAmount.String(),
-		DestAmount:           decimal.NewFromBigInt(request.Transaction.DestAmount, int32(request.DestTokenDecimals)),
+		DestAmount:           decimal.NewFromBigInt(request.Transaction.DestAmount, int32(nativeTokenDecimals)),
+		OriginFeeAmount:      decimal.NewFromBigInt(request.Transaction.OriginFeeAmount, int32(request.OriginTokenDecimals)),
+		CallValue:            decimal.NewFromBigInt(request.Transaction.CallValue, int32(request.OriginTokenDecimals)),
```

Comment:
@dwasse It seems that `DestAmount` decimals was updated in 94ee810455b277ed48dfaf050c23dc4144320355, instead of `CallValue`

Reply:
Oops, great catch!